### DISCUSSION TITLE
linkerd command get error

### DISCUSSION THREAD

**Discussion by storehubdeploy** [linkerd/linkerd2#11219] [Help]
linekerd cli from 2.11 upgrade to 2.13
use linkerd command get error 
Error rendering install manifest: error configuring Kubernetes API clientset: exec plugin: invalid apiVersion "client.authentication.k8s.io/v1alpha1"
K8S version:  EKS 1.25

**Comment by adleong** [comment]
hi @storehubdeploy 

Can you provide the exact linkerd command you ran and the exact output it produced?  It would also be helpful to see the output from `kubectl version`, `linkerd version`, and `linkerd check --pre`.

**Reply by storehubdeploy** [reply]
thanks your reply
this is command error tops
kubectl version
Client Version: version.Info{Major:"1", Minor:"23+", GitVersion:"v1.23.17-eks-0a21954", GitCommit:"cd5c12c51b0899612375453f7a7c2e7b6563f5e9", GitTreeState:"clean", BuildDate:"2023-04-15T00:35:51Z", GoVersion:"go1.19.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"25+", GitVersion:"v1.25.11-eks-a5565ad", GitCommit:"df109513ba589adaee235b4e658476275d1fde31", GitTreeState:"clean", BuildDate:"2023-06-16T17:35:00Z", GoVersion:"go1.19.10", Compiler:"gc", Platform:"linux/amd64"}
WARNING: version difference between client (1.23) and server (1.25) exceeds the supported minor version skew of +/-1

linkerd version
Error: error configuring Kubernetes API clientset: exec plugin: invalid apiVersion "client.authentication.k8s.io/v1alpha1"
Usage:
  linkerd version [flags]

Flags:
      --client             Print the client version only
  -h, --help               help for version
  -n, --namespace string   Namespace to use for --proxy versions (default: all namespaces)
      --proxy              Print data-plane versions
      --short              Print the version number(s) only, with no additional output

Global Flags:
      --api-addr string            Override kubeconfig and communicate directly with the control plane at host:port (mostly for testing)
      --as string                  Username to impersonate for Kubernetes operations
      --as-group stringArray       Group to impersonate for Kubernetes operations
      --cni-namespace string       Namespace in which the Linkerd CNI plugin is installed (default "linkerd-cni")
      --context string             Name of the kubeconfig context to use
      --kubeconfig string          Path to the kubeconfig file to use for CLI requests
  -L, --linkerd-namespace string   Namespace in which Linkerd is installed ($LINKERD_NAMESPACE) (default "linkerd")
      --verbose                    Turn on debug logging

**Reply by alpeb** [reply]
Most likely these problems are caused by the versions skew: kubectl v1.25 doesn't support a v1.23 cluster. Also linkerd upgrades are only supported one major version at a time (first 2.11 to 2.12, then 2.12 to 2.13 in two separate steps).
